packages/wdio-applitools-service/README.md

```diff
@@ -60,6 +60,11 @@ describe('My Google Search', () => {
         browser.keys('Enter')
         browser.takeSnapshot('search')
     })
+    
+    it('should open the page and take snapshot of the region with reddit icon in upper left', () => {
+        browser.url('https://reddit.com')
+        browser.takeRegionSnapshot('Reddit icon; main page', Target.region('css=a._30BbATRhFv3V83DHNDjJAO'))
```

Comment:
Can we add the import for `Target`

Reply:
Do we still need the import for Target here if we are abstracting the use of Target away from consumers of the package?